p2p/src/swarm/mod.rs

```diff
@@ -253,6 +266,27 @@ where
         Ok(())
     }
 
+    /// Adjust peer score
+    ///
+    /// If after adjustment the peer score is more than the ban threshold, the peer is banned.
+    async fn adjust_peer_score(&mut self, peer_id: T::PeerId, score: u32) -> crate::Result<()> {
+        log::debug!("adjusting score for peer {}, adjustment {}", peer_id, score);
+
+        let score = if let Some(entry) = self.peers.get_mut(&peer_id) {
+            entry.score = entry.score.saturating_add(score);
+            entry.score
+        } else {
+            score
+        };
+
+        if score >= 100 {
```

Comment:
This should come from a configuration parameter, not a constant.

Reply:
I'll add a todo